kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Network/TabTcpIpUi.java

```diff
@@ -378,21 +380,17 @@ private void initForm() {
         }
 
         // Populate status list
+        this.status.clear();
         if (this.selectedNetIfConfig != null && this.selectedNetIfConfig.getHwTypeEnum() == GwtNetIfType.MODEM) {
-            if (this.status != null) {
```

Comment:
@MMaiero do we need to remove other 'this.status == null' checks as well? (e.g. in the isLanEnabled() and isWanEnabled() methods) Can @UiField be null ?